backend/backend/controllers/api/v1.py

```diff
@@ -16,5 +18,14 @@ def handle_query_for_dashboard(self, input_sentence, subgraph):
         gql = ai_service.request_gql_for_graph(input_sentence, subgraph)
         graph_service = GraphService(protocol=subgraph)
         result = graph_service.query_thegraph(gql)
+        dashboard_result = DashboardQueryResult(user_input=input_sentence,
+                                                subgraph=subgraph,
+                                                chatgpt_gql=gql,
+                                                gql_valid=(result != ""))
```

Comment:
i understand this is not implemented yet in the `to_dict`, so it doesnt matter yet.

but i think we need to make bad results from the graphapi explicit, ie `return -1` or something

added notes to #17

Reply:
Yeah 100% agree there should be a better signaling mechanism.